Diploma/PageObjects/CheckoutPage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Playwright;
+
+namespace Diploma.PageObjects
+{
+    internal class CheckoutPage(IPage? page)
+    {
+        // Locators
+        private ILocator Checkout => page!.GetByText("Proceed To Checkout");
+
+        private ILocator AddressDetails => page!.GetByText("Address Details");
+        private ILocator ReviewYourOrder => page!.GetByText("Review Your Order");
+
+        private ILocator CommentInput => page!.Locator("//textarea[@name='message']");
+        
+
+        // Methods
+        public async Task MakeCheckout()
+        {
+            await Checkout.ClickAsync();
+
+            await AddressDetails.IsVisibleAsync();
```

Comment:
IsVisibleAsync does not wait for element. It just returns true or false (this result is not used), so regardless if this element is visible or not, code execution will go to the next line. It is like to have code
2 + 2;
"a" == "b";
you need to use WaitForAsync()